### PR TITLE
Update content for nudge unsubmitted applications email

### PR DESCRIPTION
**Pull request by stevehook**
## Context

This PR just makes a few small content changes to the email template for unsubmitted candidate nudges. It relates to https://github.com/DFE-Digital/apply-for-teacher-training/pull/6725

### Before
image

### After
image

## Guidance to review

- Does the new content look right?

## Link to Trello card

https://trello.com/c/QoHcxwpi/4594-update-email-template-for-nudges-to-un-submitted-candidates

## Things to check

- [x] If the code removes any existing feature flags, a data migration has also been added to delete the entry from the database
- [x] This code does not rely on migrations in the same Pull Request
- [x] If this code includes a migration adding or changing columns, it also backfills existing records for consistency
- [x] API release notes have been updated if necessary
- [x] Required environment variables have been updated [added to the Azure KeyVault](/docs/environment-variables.md#deploy-pipeline)

### CODE REVIEW

**Review by EmmaFrith — APPROVED** [review]
thanks Steve!

File: app/views/candidate_mailer/nudge_unsubmitted.text.erb [hunk 1]
```diff
@@ -2,6 +2,10 @@ Dear <%= @application_form.first_name %>
 
 It looks like your teacher training application is ready to submit.
 
+If you’re ready to submit now, sign in:
```
Comment (EmmaFrith) [inline app/views/candidate_mailer/nudge_unsubmitted.text.erb:5]:
```suggestion
If you’re ready to submit now, sign in to your account:
```

File: app/views/candidate_mailer/nudge_unsubmitted.text.erb [hunk 2]
```diff
@@ -2,6 +2,10 @@ Dear <%= @application_form.first_name %>
...
+
+<%= candidate_magic_link(@application_form.candidate) %>
+
 If something’s holding you back from submitting your application, we may be able to help.
 
 Do you have a teacher training adviser yet? They can help you ensure that your application is as good as it can be.
```
Comment (EmmaFrith) [inline app/views/candidate_mailer/nudge_unsubmitted.text.erb:11]:
```suggestion
Do you have a teacher training adviser yet? They can help you ensure that your application is ready to submit.
```

**Review by tomas-stefano — APPROVED** [review]
🚀

**Pull request merged**